tests/share/test_packager.py

```diff
@@ -224,3 +248,78 @@ def test_iterator_view_round_trip(self, reset_db: None) -> None:
         assert v_replica.count() == snapshot_row_count
         t_replica = v_replica.base_table
         assert t_replica.count() == 2
+
+    def test_multi_view_round_trip_1(self, reset_db: None) -> None:
+        """
+        Simplest multi-view test: two snapshots that are exported at the same time.
+        (All v_min/v_max values are consistent in the bundles.)
+        """
+
+        t = pxt.create_table('base_tbl', {'int_col': pxt.Int})
```

Comment:
Why not use the standard test table schema, or something else with more columns and types? I'm concerned about missing corner cases.